apps/site-app/src/components/Footer/index.tsx

```diff
@@ -5,7 +5,7 @@ import styles from './styles.module.less';
 
 export const Footer = () => {
   return (
-    <div id={styles.footerContainer} className={styles.desktopContent}>
```

Comment:
Didn't find any css for that class anywhere. Removing it fixes the footer problem.

Reply:
our churn lets in a lot of bugs. Adding storybook to our testing will help with this.

Reply:
Should this be merged?

Reply:
yes of course, I generally leave the PR for the author to merge when their happy.  I'll merge this one in now tho!